react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import AKButton from '@atlaskit/button';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog failure.
+ *
+ * @param {Object} props - The properties for the inline dialog.
+ * @returns {XML}
```

Comment:
->A Component's render method returns a React element not XML
Absolutely I haven't noticed that the autogenerated doc writes XML..
-> alpha order methods, exports, imports, etc, etc
Apart from the react and atlaskit import do you refer to something else with this one?